gwpy/astro/range.py

```diff
@@ -34,7 +34,7 @@
 __author__ = 'Duncan Macleod <duncan.macleod@ligo.org>'
 __credits__ = 'Alex Urban <alexander.urban@ligo.org>'
 
-DEFAULT_FFT_METHOD = None
+DEFAULT_FFT_METHOD = "median"
```

Comment:
@alurban, why is this repeated here and not just an import of `gwpy.timeseries.timeseries.DEFAULT_FFT_METHOD`?

Reply:
Ping @alurban.